app/assets/javascripts/validation.js

```diff
@@ -183,7 +192,12 @@ function validateConfirmPassword(passwordElement, obj) {
   var confirmPassword = this.value;
   var password = passwordElement.value;
 
-  if (confirmPassword !== password) {
+  if (confirmPassword === "") {
```

Comment:
And here

Reply:
Yeah, I've noticed that I've missed it after I pushed.